setup.py

```diff
@@ -99,8 +99,6 @@
     'django-reversion>=2,<3',
     'django-timezone-field>=1.2,<3',
     'djangorestframework>=3.5,<3.6',
```

Comment:
So what is this needed for anymore?

Reply:
this is needed for GDPR, unfortunately. but it only use serializers and it shouldn't block bumping DRF